joins/operations/default.json

```diff
@@ -0,0 +1,243 @@
+      "query": "FROM join_base_idx | lookup join lookup_idx_1000000_f10 on key_1000000 | where concat(lookup_keyword_0, \"foo\") == \"bar\""
+    },
+
+
+    {
+      "name": "esql_lookup_join_5M_keys_limit1",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_5000000_f10 on key_5000000 | limit 1"
+    },
+    {
+      "name": "esql_lookup_join_5M_keys_limit1000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_5000000_f10 on key_5000000 | limit 1000"
+    },
+    {
+      "name": "esql_lookup_join_5M_keys_limit10000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_5000000_f10 on key_5000000 | limit 10000"
+    },
+    {
+      "name": "esql_lookup_join_5M_keys_where_limit1000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_5000000_f10 on key_5000000 | where lookup_keyword_0 like \"val 1*\" | limit 1000"
+    },
+    {
+      "name": "esql_lookup_join_5M_keys_where_no_match",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_5000000_f10 on key_5000000 | where concat(lookup_keyword_0, \"foo\") == \"bar\""
+    },
+
+
+    {
+      "name": "esql_lookup_join_100M_keys_limit1",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_100000000_f10 on key_100000000 | limit 1"
+    },
+    {
+      "name": "esql_lookup_join_100M_keys_limit1000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_100000000_f10 on key_100000000 | limit 1000"
+    },
+    {
+      "name": "esql_lookup_join_100M_keys_limit10000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_100000000_f10 on key_100000000 | limit 10000"
+    },
+    {
+      "name": "esql_lookup_join_100M_keys_where_limit1000",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_100000000_f10 on key_100000000 | where lookup_keyword_0 like \"val 1*\" | limit 1000"
+    },
+    {
+      "name": "esql_lookup_join_100M_keys_where_no_match",
+      "operation-type": "esql",
+      "query": "FROM join_base_idx | lookup join lookup_idx_100000000_f10 on key_100000000 | where concat(lookup_keyword_0, \"foo\") == \"bar\""
+    },
+
+
+
+    {
+    "name": "esql_lookup_join_1k_100k_200k_500k",
+    "operation-type": "esql",
+    "query": "FROM join_base_idx | lookup join lookup_idx_1000_f10 on key_1000 | rename lookup_keyword_0 as lk_1k | lookup join lookup_idx_100000_f10 on key_100000 | rename lookup_keyword_0 as lk_100k | lookup join lookup_idx_200000_f10 on key_200000 | rename lookup_keyword_0 as lk_200k | lookup join lookup_idx_500000_f10 on key_500000 | rename lookup_keyword_0 as lk_500k | keep id, key_1000, key_100000, key_200000, key_500000, lk_1k, lk_100k, lk_200k, lk_500k | limit 1000"
```

Comment:
The limit 1000 at the end is pushed down, and therefore all lookups will happen on the coordinator node, I think. This means we also perform the lookups against max 1000 rows.

The same is true essentially for all other queries in this file because we add an implicit `LIMIT 1000` per default.

I think it'd be good to have versions of these queries where we add a `SORT ...` at the end - this should force the lookups onto the data nodes (best to confirm, though, by looking at the plans).

Reply:
Good point, I think it makes sense to have it as next iteration. 
I'll add some queries right away

Reply:
Actually, I realized I was being imprecise. The queries with a `WHERE` clause will not have the limit pushed down past the LOOKUP JOINs, because the WHERE clause should prevent the pushdown. So there's already queries that force the execution of LOOKUP JOINs onto the data nodes.